File: Assignment_11/program5.c
// Input 5

/*
    2       4       6       8       10      12      14      16    #       1       *       #       2       *       #       3       *       #       4       *    1       *       2       *       3       *       4       *       5       *
*/

#include<stdio.h>

int Pattern(int iNo)
{
    int iCnt = 0;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d\t",iCnt*2);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter Number of Elements : ");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

//  Time Complexity = O(N)

File: Assignment_13/program5.java
// Input 3  4

/*
    1       2       3       4
    5       6       7       8
    9       10      11      12
*/
import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0, iNo = 0;

        for(i = 1, iNo = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++, iNo++)
            {
                System.out.print(iNo+"\t");
            }
            System.out.println();
        }
    }
}

class program5
{
    public static void main(String A[])
    {
        int iValue1 = 0, iValue2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter rows and columns: ");
        iValue1 = sobj.nextInt();
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1,iValue2);
    }
}

//  Time Complexity = O(N)